OpenRA.Mods.Common/Widgets/ChatDisplayWidget.cs

```diff
@@ -105,23 +105,25 @@ public override void Tick()
 				return;
 
 			// This takes advantage of the fact that recentLines is ordered by expiration, from sooner to later
-			while (recentLines.Count > 0 && Game.LocalTick >= recentLines[0].Expiration)
-				recentLines.RemoveAt(0);
+			//while (recentLines.Count > 0 && Game.LocalTick >= recentLines[0].Expiration)
+			//	recentLines.RemoveAt(0);
```

Comment:
Any reason why that code is commented? Looks like it's still needed to me.

Reply:
Yeah i forgot to uncomment these after testing